startup.py

```diff
@@ -196,13 +195,18 @@ def _get_installation_paths_from_registry(logger):
     :returns: List of paths where 3dsmax is installed,
     """
 
-    logger.debug(
-        "Querying windows registry for key HKEY_LOCAL_MACHINE\\SOFTWARE\\Autodesk\\3dsMax"
-    )
+    if not sgtk.util.is_windows():
+        return []
```

Comment:
This part is useless. Because this function is called by `_find_software` which is called by `scan_software`.

And `scan_software`, before calling `_find_software` has the same condition.